tests/results/TextPositionTest.py

```diff
@@ -30,3 +40,58 @@ def test_properties(self):
         uut = TextPosition(8, 39)
         self.assertEqual(uut.line, 8)
         self.assertEqual(uut.column, 39)
+
+
+class TextPositionLineColTest(unittest.TestCase):
+
+    def test_from_position(self):
```

Comment:
Maybe, it'll be better to separate this into several tests. Then if something breaks, you'll know exactly what it is just from looking at test results and going through them.